src/main/java/com/derongan/minecraft/deeperworld/DeeperCommandExecutor.kt

```diff
@@ -1,62 +1,70 @@
 package com.derongan.minecraft.deeperworld
 
+import com.derongan.minecraft.deeperworld.MinecraftConstants.FULL_DAY_TIME
 import com.derongan.minecraft.deeperworld.config.DeeperConfig
 import com.derongan.minecraft.deeperworld.services.WorldManager
 import com.derongan.minecraft.deeperworld.services.canMoveSections
 import com.mineinabyss.idofront.commands.CommandHolder
+import com.mineinabyss.idofront.commands.arguments.booleanArg
 import com.mineinabyss.idofront.commands.arguments.intArg
 import com.mineinabyss.idofront.commands.execution.ExperimentalCommandDSL
 import com.mineinabyss.idofront.commands.execution.IdofrontCommandExecutor
+import com.mineinabyss.idofront.commands.execution.stopCommand
 import com.mineinabyss.idofront.commands.extensions.actions.playerAction
 import com.mineinabyss.idofront.messaging.info
 import com.mineinabyss.idofront.messaging.success
 
 @ExperimentalCommandDSL
 object DeeperCommandExecutor : IdofrontCommandExecutor() {
     override val commands: CommandHolder = commands(deeperWorld) {
-        "sectionoff" {
-            playerAction {
-                player.canMoveSections = false
-                sender.success("Automatic TP disabled for ${player.name}")
-            }
-        }
-        "sectionon" {
-            playerAction {
-                player.canMoveSections = true
-                sender.success("Automatic TP enabled for ${player.name}")
+        ("deeperworld" / "dw") {
+            "tp"(desc = "Enables or disables automatic teleports between sections for a player") {
+                val state by booleanArg()
+                playerAction {
+                    if (state) {
+                        player.canMoveSections = true
+                        sender.success("Automatic TP enabled for ${player.name}")
+                    } else {
+                        player.canMoveSections = false
+                        sender.success("Automatic TP disabled for ${player.name}")
+                    }
+                }
             }
-        }
-        "linfo" {
-            playerAction {
-                val section = WorldManager.getSectionFor(player.location)
-                if (section == null)
-                    sender.info("${player.name} is not in a managed section")
-                else
-                    sender.info("${player.name} is in section ${section.key}")
+            "linfo"(desc = "Gets information about a players section") {
+                playerAction {
+                    val section = WorldManager.getSectionFor(player.location)
+                    if (section == null)
+                        sender.info("${player.name} is not in a managed section")
+                    else
+                        sender.info("${player.name} is in section ${section.key}")
+                }
             }
-        }
-        "syncedtime" {
-            "set"{
+            "time"(desc = "<set/add>") {
```

Comment:
Don't worry about description here, if someone runs the command without extra args they'll see the message with subcommand info, which i think is good enough.

Reply:
My idea behind this was that if you run `/dw` you see <set/add> behind "time". But idm removing it.